### DISCUSSION TITLE
Support queue monitoring via heartbeats

### DISCUSSION THREAD

**Discussion by gbradley** [laravel/framework#45010] [Ideas]
Sometimes your Laravel queue will stop working and jobs will no longer be processed. The typical way of monitoring this is using a "heartbeat" or "ping" service; you schedule a job to issue an HTTP request to a dedicated endpoint, and if that endpoint doesn't receive a request after a specified time, will notify you.

This strikes me as a common enough use-case that would enhance the developer experience if added to the base framework. It could be done fairly trivially by adding some keys to `config/queue.php` and scheduling jobs accordingly.

That said, I can see this is an Envoyer feature so maybe a PR would get rejected..!

**Comment by henzeb** [comment]
Do you use Horizon? Because that sounds like a job for Horizon.
